package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ngc-omnibox",
-  "version": "0.2.2",
+  "version": "0.2.3",
```

Comment:
This is a new "feature", so technically you should be incrementing the middle number.